docs/HOWTO.md

```diff
@@ -1601,6 +1602,36 @@ sudo apt install -y vim-gtk
 
 ---
 
+### Use a subdirectory of your dotfiles repo as the root of the source state
+
+By default, chezmoi uses the root of your dotfiles repo as the root of the
+source state. If your source state contains a lot of entries in its root, then
+your dotfiles repo will in turn contain a lot of entries in its root.
+
+You can instead specify that chezmoi should read the source state from a
+subdirectory of the source directory instead by creating a file called
+`.chezmoiroot` containing the relative path the source state. This reduces the
+number of files in the root of your dotfiles repo and reduces the number of
+files that you need to explicitly exclude with `.chezmoiignore`.
```

Comment:
But if you add that paragraph about explaining chezmoignore's approach, this last sentence may not be needed anymore.

Reply:
Done (in working copy).